backend/.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  semi: false,
+  printWidth: 100,
+  singleQuote: true,
+  trailingComma: "all",
+  tabWidth: 2,
+  bracketSpacing: true
```

Comment:
trailing comma all? Did you run the linting here?

Reply:
Good catch - eslint ignores hidden files by default - do you wish to change this?